superset-frontend/package.json

```diff
@@ -138,12 +139,18 @@
     "d3-scale": "^2.1.2",
     "dom-to-image-more": "^2.10.1",
     "dom-to-pdf": "^0.3.2",
+    "echarts": "^5.4.1",
```

Comment:
i.e. `echarts` is included in the plugin-chart-echarts package, and all components that depend on ECharts should also be included in that package.
```suggestion
```